### PR TITLE
Allow active/inactive timer to be set to 0 and compare timer with wall clock

### PR DESCRIPTION
**Pull request by bluca**
See commit messages for more details about the changes.

### CODE REVIEW

**Comment by bluca** [comment]
The 0 timer is something we found some customers want, over the years.

File: src/modules/ipfix/aggregator/HashtableBuckets.h [hunk 1]
```diff
@@ -32,8 +32,8 @@ class BucketListElement;
 class HashtableBucket
 {
 public:
-	uint32_t expireTime; /**<timestamp when this bucket will expire if no new flows are added*/
-	uint32_t forceExpireTime; /**<timestamp when this bucket is forced to expire */
+	time_t expireTime; /**<timestamp when this bucket will expire if no new flows are added*/
+	time_t forceExpireTime; /**<timestamp when this bucket is forced to expire */
```
Comment (ogasser) [inline src/modules/ipfix/aggregator/HashtableBuckets.h]: Should we rename this variable to `activeExpireTime` to be consistent with IPFIX terminology?

File: src/modules/ipfix/aggregator/HashtableBuckets.h [hunk 2]
```diff
@@ -32,8 +32,8 @@ class BucketListElement;
 class HashtableBucket
 {
 public:
-	uint32_t expireTime; /**<timestamp when this bucket will expire if no new flows are added*/
-	uint32_t forceExpireTime; /**<timestamp when this bucket is forced to expire */
+	time_t expireTime; /**<timestamp when this bucket will expire if no new flows are added*/
```
Comment (ogasser) [inline src/modules/ipfix/aggregator/HashtableBuckets.h]: Should we rename this variable to `inactiveExpireTime` to be consistent with IPFIX terminology?

File: src/modules/ipfix/aggregator/AggregatorBaseCfg.cpp [hunk 3]
```diff
@@ -46,8 +46,6 @@ AggregatorBaseCfg::AggregatorBaseCfg(XMLElement* elem)
 			// get the time values or set them to '0' if they are not specified
 			maxBufferTime = getTimeInUnit("activeTimeout", SEC, 0, e);
```
Comment (ogasser) [inline src/modules/ipfix/aggregator/AggregatorBaseCfg.cpp]: Should we rename this variable to `activeTimeout` to be consistent with IPFIX terminology?

File: src/modules/ipfix/aggregator/AggregatorBaseCfg.cpp [hunk 4]
```diff
@@ -46,8 +46,6 @@ AggregatorBaseCfg::AggregatorBaseCfg(XMLElement* elem)
...
 			minBufferTime = getTimeInUnit("inactiveTimeout", SEC, 0, e);
```
Comment (ogasser) [inline src/modules/ipfix/aggregator/AggregatorBaseCfg.cpp]: Should we rename this variable to `inactiveTimeout` to be consistent with IPFIX terminology?

File: src/modules/ipfix/aggregator/HashtableBuckets.h (hunk 2 above)
Comment (bluca) [inline src/modules/ipfix/aggregator/HashtableBuckets.h]: Done all renames by grep | xargs sed. It's a big-ish diff and touches many file, let me know if it's fine for you.

**Comment by ogasser** [comment]
Looks good, thank you!

**Comment by bluca** [comment]
Thanks! Just opened one with a few more bug fixes: https://github.com/tumi8/vermont/pull/61

**Pull request merged**